materials/cSharp/delegates.md

```diff
@@ -1 +1,94 @@
-# Delegáty
\ No newline at end of file
+# Delegáty
+
+Co to je delegát?: datový typ ,který představuje odkazy na metody s konkrétním seznamem parametrů a návratovým typem. Pokud vytvoříte instanci delegátu, můžete příslušnou instanci přidružit s jakoukoli metodou s 
+    kompatibilním podpisem a návratovým typem. Metodu můžete vyvolat (nebo volat) prostřednictvím instance delegátu.
+
+    jak to může vypadat:
+    
+ ![Snímek obrazovky 2024-10-22 220453](https://github.com/user-attachments/assets/ba96ac9c-c99c-4e1a-b2d1-d675747318cb)
```

Comment:
Místo všech snímků obrazovky bych preferoval kod, který půjde kopířovat, kod jde vnořit do zpětných uvozovke (anglicky backticks) ` <- toto. A když dáš tří \`\`\`  tak vytvoříš začátek bloku kodu. Pak na konec dáš taky tři a budeš mít blok kodu.
Ještě můžeš za ty první tři dát jazyk, v našem příkladě můžeš dát csharp a bude to i barevně označovat správně slova.
https://github.com/jincheng9/markdown_supported_languages
https://www.markdownguide.org/basic-syntax/